provision-profile.yml

```diff
@@ -0,0 +1,38 @@
+---
+- hosts: all
+  gather_facts: no
+  tasks:
+  - name: Read vars from secrets file
+    include_vars: "{{ secrets_file }}"
+    tags:
+      - always
+
+- hosts: loadbalancer:php-apps
+  gather_facts: no
+  sudo: true
+  roles:
+    - common
+    - tls
+  handlers:
+    - include: roles/httpd/handlers/main.yml
+    - include: roles/nginx/handlers/main.yml
+
+- hosts: loadbalancer
```

Comment:
does profile really need a loadbalancer, or is that something that the general loadbalancer can fix?